sec2-architecture.tex

```diff
@@ -36,28 +36,62 @@ \subsubsection{Functional Architecture}\label{functional-architecture}
 \subsubsection{Deployment Architecture}\label{deployment-architecture}
 
 The LSP is designed to be deployable in multiple locations within the LSST
-Data Management System (DMS).
-The production instances are the US Data Access Center, the Chilean Data Access
-Center, the Science Validation instance in the NCSA Enclave, and the
-Commissioning Cluster instance in the Base Enclave.
-Additional instances are used for integration testing in the Integration
-environment (the Prototype Data Access Center) and for science
-pipelines developer usage in the Developer environment.
-LSP developers will
-of course instantiate components of the LSP during their own development.
+Data Management System (DMS). The production instances are the US Data
+Access Center, the Chilean Data Access Center, the Science Validation
+instance in the NCSA Enclave, and the Commissioning Cluster instance in the
+Base Enclave. Additional instances are used for integration testing in the
+Integration environment (the Prototype Data Access Center) and for science
+pipelines developer usage in the Developer environment. LSP developers
+will of course instantiate components of the LSP during their own development.
 
 The LSP is composed of multiple services that are containerized and deployed
-using Kubernetes.  The underlying provisioning of compute resources is handled
-by the Provisioning and Deployment Service component of the LSST DMS.  The
-technology to be used for provisioning in this component is being evaluated and
-may vary between instances.  vSphere, HTCondor, SLURM, and OpenStack are all
-potential alternatives.  Provisioning is elastic: JupyterLab dynamically
-instantiates notebooks on demand, and the Portal and Web API services can be
-expanded or contracted as needed.
-This elasticity, together with fault tolerance considerations, require that
-the services hold minimal non-persistent state so
-that loss of a single service instance has correspondingly minimal impact on
-users.
+using Kubernetes.  The underlying provisioning of compute resources is
+handled by the Provisioning and Deployment Service component of the LSST DMS.
+ The technology to be used for provisioning in this component is being
+evaluated and may vary between instances.  vSphere, HTCondor, SLURM, and
+OpenStack are all potential alternatives.  Provisioning is elastic: JupyterLab
```

Comment:
I've always been under the impression that the compute resources would be managed HTCondor.  #1) I assumed that the list was there because no official (that I'm aware of) decision has been made about that.  #2) Additionally, while it's possible to have compute resources managed by HTCondor running in a container managed my Kubernetes, I believe it's still up for grabs about whether this is the most practical way of doing it.  It may be that shifting resources out of an idle HTCondor cluster to a Kubernetes cluster (or visa versa) may better way of doing things, given that the per job requirements of HTCondor jobs would be more difficult in a previously provisioned and running container, vs having a whole node dedicated on start up, which would have flexibility.    The transition of a machine between these two would be slower, but the compute resource allocation would be much more flexible, especially given the varying RAM and core requirements at different points in the jobs.   Since neither 1 or 2 has been officially decided, I left that as it was.